bluesky/run_engine.py

```diff
@@ -1430,7 +1434,8 @@ def _close_run(self, msg):
                    num_events=num_events)
         self._clear_run_cache()
         yield from self.emit(DocumentNames.stop, doc)
-        self.log.debug("Emitted RunStop (uid=%r)", doc['uid'])
+        doc_logger.debug("Emitted RunStop (uid=%r)", doc['uid'],
+                         extra={'doc_name': 'Stop', 'doc_uid': doc['uid']})
```

Comment:
```suggestion
                         extra={'doc_name': 'stop', 'doc_uid': doc['uid']})
```

Reply:
If so, tags of document and tags of Msg are not esay to figure out